### ISSUE TITLE
[Quest][Howling Fjord] The Clutches of Evil quest not showing up

### ISSUE THREAD

**Issue by Uziel25** [dalaranwow/dalaran-wow#5350]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When visiting "Captain Adams" in Howling Fjord he does NOT have the quest "Clutches of Evil".  See http://www.wowhead.com/quest=11157/the-clutches-of-evil
**Expected behaviour**: Tell us what should happen instead
He is supposed to have this quest for all players with no prequests required.
**Steps to reproduce the problem**:

1. Visit Capain Adams with any character level 58 or higher.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
http://www.wowhead.com/quest=11157/the-clutches-of-evil is the wowhead page.
https://github.com/Atlantiss/BugTracker/issues/2585 is the same bug on the Atlantiss server.
https://community.atlantiss.eu/index.php?/topic/15106-quest-the-clutches-of-evil/ is how they fixed it on their server.  It may be possible that this fix may be usable on Dalaran wow.
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
No prequest is required. Currently xou need to do http://www.wowhead.com/quest=11501/news-from-the-east#comments first in order to get https://wowgaming.altervista.org/aowow/?quest=11157.

That#s wrong according to this video https://www.youtube.com/watch?v=JE80Vhj2zeI&feature=youtu.be&t=2m4s and that text:
